reconstruction/cvt/src/main/java/org/jlab/rec/cvt/services/TracksFromTargetRec.java

```diff
@@ -191,15 +191,19 @@ public List<Track> getTracks(DataEvent event, boolean initFromMc, boolean kfFilt
         kf2.polarity = (int) Math.signum(Constants.getSolenoidScale());
         Measurements measure = new Measurements(xb, yb, Constants.getInstance().kfBeamSpotConstraint());
         for (Seed seed : this.CVTseeds) { 
+            measure.setSeed(seed);
```

Comment:
this should not be necessary because seed is the argument of getMeasurement at line 199

Reply:
no longer necessary after changes implemented in comment above.  So it is removed.